include/bl_crypto.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef BOOTLOADER_CRYPTO_H__
+#define BOOTLOADER_CRYPTO_H__
+
+#include <zephyr/types.h>
+#include <fw_metadata.h>
+
+/* Placeholder defines. Values should be updated, if no existing errors can be
+ * used instead. */
+#define EHASHINV 101
+#define ESIGINV  102
+
+
+#if CONFIG_SB_CRYPTO_OBERON_SHA256
+	#include <occ_sha256.h>
```

Comment:
Don't indent pre-compiler directives, if possible.

Reply:
It's totally unreadable without, so pretty please?

Reply:
Ok, so can we merge @lemrey ?